capstone-frontend/app.yaml

```diff
@@ -1,11 +1,22 @@
-runtime: nodejs
-env: flex
+runtime: nodejs12
+service: reactapp # Name of Service, instead of default
```

Comment:
I saw you changed this back to default in the next PR, why did you change it back and forth? And did you intend to switch it back to "default" later?

Reply:
I did the change to change the website url. Instead of reactApp-mauroguerrero-step2020 is going to be just mauroguerrero-step2020.